java-manta-it/src/test/java/com/joyent/manta/client/MantaClientIT.java

```diff
@@ -400,8 +426,7 @@ public final void canMoveEmptyDirectory() throws IOException {
             + path);
     }
 
-
-    public void moveDirectoryWithContents(final String source, final String destination) throws IOException {
+    private void moveDirectoryWithContents(final String source, final String destination) throws IOException {
```

Comment:
This was originally a mistake to code this method as public. In fact, it was being invoked by the test runner and then subsequently skipped.